Frameworks/UIKit/UIButton.mm

```diff
@@ -891,22 +946,18 @@ - (UIImageView*)imageView {
 }
 
 /**
- @Status Caveat
- @Notes Returns the receiving view
-*/
-- (UIView*)viewForBaselineLayout {
-    return self;
-}
-
-/**
  @Status Interoperable
 */
 - (CGSize)intrinsicContentSize {
     CGSize ret = CGSizeZero;
 
     UIImage* img = self.currentImage;
     UIEdgeInsets contentInsets = self.contentEdgeInsets;
-    CGSize textSize = [[self currentTitle] sizeWithFont:self.font];
+
+    // TODO: Should we be asking our label for its intrinsicContentSize?
```

Comment:
wish the doc is more clear, looks good candidate for an experiment later.

---
In reply to: [100653270](https://github.com/Microsoft/WinObjC/pull/1920#discussion_r100653270) [](ancestors = 100653270,100653027)